src/styles/main.scss

```
@@ -1,3 +1,67 @@
+@import "utils/variables";
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
```

" * працює на всі об'єкти, тому немає сенсу задавати додаткові селектори.
```suggestion
* {
  box-sizing: border-box;
}
```